homeassistant/components/tado/manifest.json

```diff
@@ -14,5 +14,5 @@
   },
   "iot_class": "cloud_polling",
   "loggers": ["PyTado"],
-  "requirements": ["python-tado==0.17.6"]
+  "requirements": ["python-tado-ha==0.1.3"]
```

Comment:
I'd advise against using ha in the name if that refers to Home Assistant. We want our 3rd party libraries to be useful for the whole open source community and not exclude other usage needlessly. Consider renaming the package.

Reply:
The PR description is confusing since we're still using a 3rd party library.

>  It gives direct control within the HA community and no third-party library is used anymore.

This library is not the standard library and not Home Assistant, so it's 3rd party.

Reply:
> I'd advise against using ha in the name if that refers to Home Assistant. We want our 3rd party libraries to be useful for the whole open source community and not exclude other usage needlessly. Consider renaming the package.

I can relate to that. It shouldn't give the impression it's associated as an integral part of HA, though the current scope is limited to endpoints only used by HA.
I'll rename it to something more generic.